lib/Rex/Commands/File.pm

```diff
@@ -314,7 +314,7 @@ Furthermore, if a path prefix matches multiple prefix entries in 'path_map',
 e.g. "files/etc/ntpd.conf" matching both "files/" and "files/etc/", the
 longer matching prefix(es) have precedence over shorter ones. Note that
 keys without a trailing slash (i.e. "files/etc") will be treated as having
-a trailing slash when matching the prefix ("files/etc/"). 
+a trailing slash when matching the prefix ("files/etc/").
```

Comment:
No, if this is removed, pod will think that the example code doesn't belong together. those spaces are needed.